c2corg_ui/static/img/fonts/README.md

```diff
@@ -26,6 +26,7 @@ to
     url('fonts/heavy_metal.woff?56tqll') format('woff'),
     url('fonts/heavy_metal.svg?56tqll#icomoon') format('svg');```
 
+9) @import 'heavymetal.less'; in c2corg_ui.less
```

Comment:
What's this used for?
I see no mention to ``heavymetal.less`` anywhere. And why don't you add this ``@import 'heavymetal.less';`` to ``c2corg_ui.less`` directly anyway?